docs/build_pipeline_controller.md

```diff
@@ -24,15 +25,20 @@ add_finalizer(Add finalizer to build PLR)
 remove_finalizer(Remove finalizer from build PLR)
 error[Return error]
 continue[Continue processing]
+update_medata(add PR group info to build pipelineRun metadata)
```

Comment:
```suggestion
update_metadata(add PR group info to build pipelineRun metadata)
```

Reply:
Thanks. Updated.

Reply:
Actually, seems like it reverted to `update_medata`  with the last commit

Reply:
Thanks! Updated again.